tooling/api/src/helpers/env-check.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019-2021 Tauri Programme within The Commons Conservancy
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: MIT
+
+/** @ignore */
+
+function isBrowser(): boolean {
+  return typeof window !== 'undefined'
+}
+
+function isLinux(): boolean {
+  if (isBrowser()) {
+    return navigator.appVersion.includes('Linux')
+  } else {
+    return process.platform === 'linux'
+  }
+}
```

Comment:
This is useless and doesn't make sense at all. `isLinux()` and siblings are checks that is meant to be used in runtime only and not evaluated at compile/bundle time. And `@tauri-apps/api` should only work in browser environment.

Reply:
It's not for compile time, it's to make it actually be possible to import the package in SSR and as a result make errors easier to debug

Reply:
So if your SSR server is running on Linux, `isLinux()` will always return true while the app actually runs on Windows or macOS.

Reply:
I understand what you want to achieve but this isn't the way. Instead we need to wrap all our apis under some guards so imports won't break.

Reply:
Then `sep` and `delimiter` would have to become functions

Reply:
I don't like that right now, it is a breaking change. And as I always says, we can only do so little about SSR. People shouldn't use SSR with tauri, if they do then it is their job to fix any quirks about it.

Reply:
I agree, and I'm not sure they should ever become functions either

Reply:
Yeah, they shouldn't. SSR users should deal with this on their own and just use async imports.

Reply:
Why, what's the issue with having it be possible to import it in this way?